BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -259,37 +325,27 @@ namespace big
 
 		if (ImGui::Checkbox("Headlight##headlight_en", &xenon))
 		{
-			g_fiber_pool->queue_job([] {
-				VEHICLE::TOGGLE_VEHICLE_MOD(player_vehicle, MOD_XENON_LIGHTS, xenon);
-			});
+			VEHICLE::TOGGLE_VEHICLE_MOD(player_vehicle, MOD_XENON_LIGHTS, xenon);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.